mkosi/resources/mkosi.md

```diff
@@ -1450,6 +1438,26 @@ boolean argument: either `1`, `yes`, or `true` to enable, or `0`, `no`,
 : List of colon-separated paths to look for tools in, before using the
   regular `$PATH` search path.
 
+`Vmm=`, `--vmm=`
```

Comment:
Though explained in the next sentence, the name is a bit cryptic. Maybe `VmMonitor=`?

Reply:
Vmm seems to be the usual abbrevation used for this. I want to rename all `QemuXXX=` settings to `VmmXXX=` in the future, so keeping it short seems good here?

Reply:
Renamed to `VirtualMachineMonitor=` but kept `--vmm`

Reply:
With that consideration in mind that seems seems sensible.